packages/x-domain/prettier-config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Is there any way that this could be part of `@eth-optimism/dev`?

Reply:
Possibly yes, but I haven't looked into the implications of doing it this way.

Reply:
I'll add a ticket for it on `@eth-optimism/dev` and handle this in another PR.

Reply:
NVM -- just fixed this here instead :-)